schema/Claim.schema.yaml

```
@@ -0,0 +1,41 @@
+title: Claim
+'@id': stencila:Claim
+extends: Entity
+role: secondary
+status: unstable
+description: A claim represents specific reviewable facts or statements.
+$comment: |
+  It is usually displayed as a block element and can have a separate title.
+  In the context of a scholarly article this could be a theorem or proof.
+  See also
+  - [Schema.org `Claim`](https://schema.org/Claim)
+  - [JATS `<statement>`](https://jats.nlm.nih.gov/publishing/tag-library/1.2/element/statement.html)
+properties:
+  title:
+    '@id': stencila:title
+    description: Title of the claim.
+    type: string
+  claimType:
+    '@id': stencila:claimType
+    description: Kind of the claim.
+    type: string
+    enum:
+      - Claim
+      - Theorem
+      - Lemma
+      - Proof
+      - Postulate
+      - Hypothesis
+      - Proposition
+      - Corollary
```

JATS `<statement>` element has the content model `(label?, title?, (abstract)*, (kwd-group)*, (p | statement)+, (attrib | permissions)*)`. I suggest adding `label` (as we have for `Figure`, `Table`, `MathBlock`). `title` & `abstract` (aka `description`) and `keywords` are already provided in `CreativeWork`.

```suggestion
      - Corollary
    label:
      '@id': stencila:label
      description: A short label for the claim.
      type: string
```